src/overlays/actors/ovl_En_Syateki_Niw/z_en_syateki_niw.c

```diff
@@ -477,93 +476,92 @@ void func_80B12460(EnSyatekiNiw* this, PlayState* play) {
                        RAD_TO_BINANG(Math_FAtan2F(player->actor.world.pos.x - this->actor.world.pos.x,
                                                   player->actor.world.pos.z - this->actor.world.pos.z)) +
                            phi_f16,
-                       5, this->unk_2C8.y, 0);
-    Math_ApproachF(&this->unk_2C8.y, 3000.0f, 1.0f, 500.0f);
-    if (this->unk_296 == 2) {
-        this->unk_256 = 10;
-        this->unk_254 = this->unk_256;
+                       5, this->posRotStep.y, 0);
+    Math_ApproachF(&this->posRotStep.y, 3000.0f, 1.0f, 500.0f);
+    if (this->archeryAnimationType == 2) {
+        this->peckTimer = this->timer1 = 10;
     }
 
-    func_80B11A94(this, play, this->unk_296);
+    EnSyatekiNiw_UpdateRotateTargets(this, play, this->archeryAnimationType);
 }
 
-void func_80B128D8(EnSyatekiNiw* this, PlayState* play) {
-    if (this->unk_25E == 1) {
+void EnSyatekiNiw_ExitArchery(EnSyatekiNiw* this, PlayState* play) {
+    if (this->movementTimer == 1) {
         gSaveContext.timerState = TIMER_STATE_OFF;
     }
 }
 
-void func_80B128F8(EnSyatekiNiw* this, PlayState* play) {
+void EnSyatekiNiw_SetupRemove(EnSyatekiNiw* this, PlayState* play) {
     s16 sp26;
     s16 sp24;
```

Comment:
```suggestion
    s16 x;
    s16 y;
```
generally, I've just seen them named like this. Though you do use `screenX` and `screenY` down below 🤔.